biopet-tools/src/main/scala/nl/lumc/sasc/biopet/tools/ReplaceContigsGtfFile.scala

```diff
@@ -0,0 +1,85 @@
+package nl.lumc.sasc.biopet.tools
+
+import java.io.{File, PrintWriter}
+
+import nl.lumc.sasc.biopet.utils.ToolCommand
+import nl.lumc.sasc.biopet.utils.annotation.Feature
+
+import scala.io.Source
+
+/**
+  * Created by pjvan_thof on 30-5-17.
+  */
+object ReplaceContigsGtfFile extends ToolCommand {
+  case class Args(input: File = null,
+                  output: File = null,
+                  contigs: Map[String, String] = Map(),
+                  writeAsGff: Boolean = false)
+      extends AbstractArgs
+
+  class OptParser extends AbstractOptParser {
+    opt[File]('I', "input") required () valueName "<file>" action { (x, c) =>
+      c.copy(input = x)
```

Comment:
Depends who did make the tool. The standard here I copied from GATK back then but there are also tools where we do this differently.
For now this can stay like this but next meeting we can discuss a plan for this to standardize this. (will be next week)